src/dll/Systems/ScriptCompilationSystem.cpp

```diff
@@ -90,10 +90,10 @@ std::wstring ScriptCompilationSystem::GetCompilationArgs(const FixedWString& aOr
     for (const auto& [plugin, path] : m_scriptPaths)
     {
         spdlog::info(L"{}: '{}'", plugin->GetName(), path);
-        pathsFile << path << std::endl;
+        pathsFile << path.wstring() << std::endl;
```

Comment:
Why this change?

Reply:
This prevents the paths from being escaped into strings with quotes (and `\\` all over the paths) - scc is able to read the paths alone in the file, and it's easier to handle when debugging.